### DISCUSSION TITLE
You have been blocked from creating an account

### DISCUSSION THREAD

**Discussion by masumali23020** [vercel/community#3177] [Feature Question] [answered]
### Question

how to solve this issue, i can not log in or signup Vercel acount

**Comment by amyegan** [comment, marked as answer]
Please email [registration@vercel.com](mailto:registration@vercel.com) and include the email, phone number, and username used for account creation. To protect against platform abuse, Vercel employs multiple layers of security and verification. Providing additional information about your account and the project you are deploying can help accelerate this process.
